apps/pie-docs/src/components/template/template.md

```diff
@@ -0,0 +1,58 @@
+---
+eleventyNavigation:
+    key: '[Component name]'
+    parent: Components
+    order: 1
+---
+
+## Overview
+[Component Overview]
+
+
+{% contentPageImage {
+    src:""
+} %}
+
+---
+
+## Resources
+
+{% notification {
+  type: "warning",
+  message: "We’re currently working on updating our [Component name] documentation, please see the resources below."
+} %}
+
+{% resourceTable {
+    rows: [
+        {
+            resource: resourceTypes.COMPONENT,
+            link: "google.co.uk",
```

Comment:
And I know it's only a draft but just thought I'd let you know this links to `http://localhost:8080/components/template/google.co.uk` rather than `https://www.google.co.uk` so not sure if it's just the missing `https` that gets misinterpreted or if the link logic isn't quite right.

Reply:
Ahh thanks!